### ISSUE TITLE
Add list of unused modversions on dashboard

### ISSUE THREAD

**Issue by GibbsFrance** [TechnicPack/TechnicSolder#318]
Hi,

I noticed that after creating several builds, one can quickly end up with versions of mods that no longer serve any modpack.

The idea is to display a list of mods to inform the user that their removal is safe and that it can also remove the hard drive.

I did a fork and a commit (42284bb811631e798cea36d84a19000f00beff9d) that implements this functionality. However, I find that the code in the controller is not the cleanest that can exist, so if there is an easier way that would be practical.

Good day.

**Comment by GenPage** [comment]
Its a good idea, but we should get some more input on this. I dont want to clutter up the dashboard too much

**Comment by GibbsFrance** [comment]
Indeed, the dashboard is not necessarily the best place to put this kind of data. Perhaps below the list of mods (`/mod/list`)?

**Comment by TheDoctorsLife** [comment]
hmm, maybe a warning like "You have 5 unused versions of modx" when you visit a mod's page?

**Comment by GibbsFrance** [comment]
Interesting solution and probably easier to implement (I did not yet look at the code to implement this solution).

However, the user may not see the information, but this can be a good compromise.

**Comment by mcmielleman** [comment]
But i see no scenario where i would use it

**Comment by GibbsFrance** [comment]
I had the need when I have prepared a new build of a modpack. I usually check the mods, and download their updates and finally I test the build. Sometimes a mod is again updated during this period and so I will update the build. Therefore, the previous update of the mods is no longer used.

Also, after a while, the creator of a modpack may well remove a build. We can then have versions of mods that will not be used by any other build.

**Comment by FlareHAX0R** [comment]
or possibly Mod>Notifications

btw, 1+ on this :P

**Comment by RAWRwins254** [comment]
+1 on this. However, I would say do a small block notification that the user can click on and be redirected to a page solely devoted to the mods not being used anywhere.
